src/lib/wfdbio.cc

```diff
@@ -155,6 +155,8 @@ obtained if necessary by defining the symbol NOSTRTOK when compiling this
 module.
 */
 
+#include <stdlib.h>
```

Comment:
https://google.github.io/styleguide/cppguide.html#Include_What_You_Use

Discourage use of transitive dependencies.